src/ncp/ncp_openthread.cpp

```diff
@@ -215,6 +215,11 @@ void ControllerOpenThread::Init(void)
         VerifyOrExit(result == OT_ERROR_NONE, error = OTBR_ERROR_OPENTHREAD);
     }
 
+#if OTBR_ENABLE_FEATURE_FLAGS && OTBR_ENABLE_TREL
+    // Disable trel when feature flag is enabled, let feature flag handle enable/disable trel
+    otTrelSetEnabled(mInstance, false);
```

Comment:
Is it possible to get here with feature flags wanting the TREL to be enabled already?

Reply:
Good point, the feature flags are updated by thread daemon after otbr-agent signals OTBR_DBUS_SIGNAL_READY, so here we don't get the updated value. But the default feature flag value can be 'true' or 'false' so I think we should use that instead. Will update.